<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consulList</class>
 <widget class="QWidget" name="consulList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background:red;
}

QFrame#FrameBase
{
background:#FFFBFF;
border-radius: 16px;
}

QFrame#mainFrame
{
background:rgba(83, 85, 168, 20);

border-radius: 16px;

}

QFrame#buttonFrame
{
background:none;
}

QFrame#dataFrame
{
background:None;

border-bottom-width: 1px;
border-style: solid;
border-bottom-color: #E4E1EC;
}

QFrame#dataButtonFrame
{
background:none;
}

QLabel#iconLabel
{
background:None;
color:#5355A8;
font-family:Material Icons;
font-style: normal;
font-size: 26px;
}

QLineEdit#searchEdit {
color:#5355A8;
font-family: 'Roboto';
background: transparent;
border:none;
font-weight: 500;
font-size: 14px;
line-height: 20px;
}

QScrollArea#scrollAreaList
{
background:transparent;
border:None;
}

QScrollBar:vertical
{
background:None;
border:None;
width: 0px;
}

QWidget#listScroll
{
background:transparent;

}

QFrame#itemFrame
{
background:none;
}

QLabel
{
background:None;
}

</string>
  </property>
  <widget class="QFrame" name="FrameBase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>245</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>125</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>245</width>
     <height>400</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="dataFrame">
         <property name="maximumSize">
          <size>
           <width>245</width>
           <height>334</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QScrollArea" name="scrollAreaList">
          <property name="geometry">
           <rect>
            <x>3</x>
            <y>9</y>
            <width>239</width>
            <height>331</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>239</width>
            <height>331</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="listScroll">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>239</width>
             <height>331</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="optionsLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="itemFrame">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>239</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>38</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="buttonFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>56</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>56</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="dataButtonFrame">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>8</y>
            <width>209</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>89</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>209</width>
            <height>40</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="iconLabel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>40</width>
             <height>40</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>î……</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="searchEdit">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>10</y>
             <width>160</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Nueva</string>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
